src/mx/NotificationManager.java

```diff
@@ -0,0 +1,24 @@
+package mx;
+
+import java.util.ArrayList;
+
+/**
+ * Created by masterix on 28.05.2017.
+ */
+public class NotificationManager {
+
+    private ArrayList<NotificationSenderInterface> senders;
```

Comment:
It's better practice to rely on abstractions (here: interface), not implementation details. so it should be `private List<NotificationSender> senders;`